src/Symfony/Component/ImportMaps/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2023 Kévin Dunglas
```

Comment:
Should this line be in line with this commit? `Copyright (c) 2023-present Fabien Potencier` https://github.com/symfony/symfony/commit/d2e034abf016f4da9a840d17af64a6573353ae9?